third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1494,6 +1494,58 @@ struct ExpOpConversionApprox
   }
 };
 
+class ClampFOpConversion : public ClampFOpConversionBase {
+public:
+  // Constructor
+  explicit ClampFOpConversion(LLVMTypeConverter &typeConverter,
+                              ModuleAxisInfoAnalysis &axisAnalysisPass,
+                              int computeCapability, PatternBenefit benefit = 1)
+      : ClampFOpConversionBase(typeConverter, axisAnalysisPass,
+                               computeCapability, benefit) {}
+
+  bool optimizedClampPatternFound(ClampFOp op) const override { return false; }
+
+  SmallVector<Value> emitOptimization(ClampFOp op,
+                                      ConversionPatternRewriter &rewriter,
+                                      Type elemTy,
+                                      MultipleOperandsRange operands,
+                                      Location loc) const override {
+    return {};
+  }
+
+  SmallVector<Value> emitDefault(ClampFOp op,
+                                 ConversionPatternRewriter &rewriter,
+                                 Type elemTy, MultipleOperandsRange operands,
+                                 Location loc) const override {
+    // Clip pattern not found, use min/max.
+    if (op.getPropagateNan() == PropagateNan::ALL) {
+      // if (computeCapability >= 80) {
```

Comment:
figure out which cc allows this on amd

Reply:
This is the same condition as the maximum/minimum workaround you added

Reply:
For now I'm disabling llvm::maximum and llvm::minimum on AMD backend. Should I keep it like this until I create the `TargetInfo` ?

Reply:
yeah or until amd patch lands in llvm

Reply:
Sounds good, this PR should be ready then.